webextension/scripts/locales.js

```diff
@@ -75,6 +76,37 @@ const localeStrings = {
 		"h1Allycss": "a11y.css",
 		"h1Allycss2": "a11y.css : réglages",
 		"h1Others": "Autres outils",
+	},
+		"pl":{
```

Comment:
Indentation feels wrong here :)

```suggestion
	"pl":{
```